src/Lachain.Core/RPC/HTTP/HttpService.cs

```diff
@@ -147,19 +159,36 @@ private bool _Handle(HttpListenerContext context)
                     response.Close();
                     return false;
                 }
-            }
-            
+            }            
 
             JsonRpcProcessor.Process(async);
             return true;
         }
 
-        private bool _CheckAuth(JObject body, HttpListenerContext context)
+        private bool _CheckAuth(JObject body, HttpListenerContext context, string sig, string bodyPlainText)
         {
+			if (string.IsNullOrEmpty(_apiKey))
```

Comment:
We don't need to check it for every request,  it is better to check it only once during HttpService start

Reply:
Removed